src/applications/personalization/dashboard/components/claims-and-appeals/ClaimsAndAppeals.jsx

```diff
@@ -266,7 +248,6 @@ const mapStateToProps = state => {
 const mapDispatchToProps = {
   loadAppeals: getAppealsAction,
   loadClaims: getClaimsAction,
-  loadLighthouseClaims: getLighthouseClaimsAction,
```

Comment:
minor comment here, I try and keep the naming the same with mapDispatchToProps across the imported actions, so when importing the action you rename `getClaims` to `getClaimsAction`, then in mapDispatchToProps you can just use the original function name:

`getClaims: getClaimsAction`

That way there aren't two different naming conventions of load vs get and it makes things a little easier to follow. Not blocking whatsoever though.

Reply:
resolved: renamed `loadAppeals` and `loadClaims` to `getAppeals` and `getClaims` in this file for consistency.